### DISCUSSION TITLE
CUDA_ERROR_OUT_OF_MEMORY

### DISCUSSION THREAD

**Discussion by robin32205** [NVlabs/instant-ngp#594] [Q&A]
hi,
When I am running testbed with NeRF dataset, I get CUDA error about out-of-memory. The reason maybe the memory I have is only 6 Gb on my GTX1060. So how can I change the config of batch size or something else to may it run well? thanks.

**Comment by gvuksic** [comment]
lower image resolution, for me personally on 1660 ti lowering it down to 1024*768 helped to start it, but GUI is super laggy, I can see render but I can hardly click anything in it, not able to set camera waypoints or to export video... results you see here on examples are from RTX 3090, compared to our cards that is much stronger...

**Comment by robin32205** [comment]
I see~ lower resolution may be a solution , but if I want to hold resolution by using more time for computing, it may have another way to achieve that... I think this must be compatible with differ NV GPU, is that right? :)
